.github/workflows/chrome-store-upload-dev.yaml

```diff
@@ -0,0 +1,31 @@
+name: Upload Dev To Store
+
+on: 
+  push:
+    branches:
+      - master
```

Comment:
Can we change this to on push to "testing" or "beta" branch? I would like to get in a cadence where we are using a merge strategy that we essentially always have our current stable release, then a beta version we can use for both beta testers who want to help the project as well as to have someone test a fix which we can't test directly. This happens a lot with subscription based video services.

This would also give us an opportunity to try out A|B testing on two styles of implementation

Reply:
I think we can get that with current strategy, no? In effect, we could treat master as tip of tree where each merge triggers a new release for beta channel. That said, one trick here is to figure out auto-increment strategy.

Reply:
I guess my thinking was that there may be some bugs we are getting fixed up in the prior release that we would want in a separate build/branch so that a bugfix release could be pushed. If we used the same branch for both it would mean that before tagging and uploading the bugfix release we would have to back out all of the beta features we had in the pipe first